src/scipp/core/variable.py

```diff
@@ -458,3 +459,86 @@ def arange(dim: str,
                  values=_np.arange(start, stop, step),
                  unit=unit,
                  dtype=dtype)
+
+
+def datetime(value: _Union[str, int, _np.datetime64],
+             *,
+             unit: _Optional[_Union[_cpp.Unit, str]] = None) -> _cpp.Variable:
+    """Constructs a zero dimensional :class:`Variable` with a dtype of datetime64.
```

Comment:
Missing `:py` before `:class`?

Reply:
Seems to be the default. I copied this from `arange` and it seems to work.